.travis.yml

```diff
@@ -1,10 +1,16 @@
+sudo: false
+addons:
+  apt:
+    packages:
+      lua5.2
 language: cpp
 env:
   matrix:
     -LUA_VERSION=5.2
 before_install:
-- sudo apt-get install lua$LUA_VERSION
-- sudo pip install docutils
+#- sudo apt-get install lua$LUA_VERSION
+#- sudo pip install docutils
+  pip install docutils
```

Comment:
You could try `pip install --user docutils` (if that's the right flag to install in $HOME)